BuildResidentialScheduleFile/measure.rb

```diff
@@ -96,6 +96,12 @@ def run(model, runner, user_arguments)
       fail "'#{hpxml_path}' does not exist or is not an .xml file."
     end
 
+    hpxml_output_path = args[:hpxml_output_path]
+    unless (Pathname.new hpxml_output_path).absolute?
+      hpxml_output_path = File.expand_path(File.join(File.dirname(__FILE__), hpxml_output_path))
+    end
+    args[:hpxml_output_path] = hpxml_output_path
```

Comment:
I know all you did was move some existing code around but... I really dislike seeing relative paths as interpreted relative to the measure itself. We need to stop doing that, it is totally confusing when the measure's location is unpredictable. I'm tempted to suggest that we should stop allowing relative paths for input/output paths in any measure. Does anyone ever really need that?

(On the other hand, for paths that end up _inside the HPXML file_, it makes complete sense to allow relative paths -- e.g., to point to a file that's in the same directory as the HPXML file itself.)

I suppose we don't have to address this now, but it might be as good a time as any.

Reply:
Right, maybe we don't need relative paths. For ResStock-HPXML, it looks like we're using absolute paths within the `BuildExistingModel` meta measure: https://github.com/NREL/resstock/blob/restructure-v3/measures/BuildExistingModel/measure.rb#L229.

Reply:
@shorowit So what should be done here? Update all comments by removing all "relative path" references? Also, have checks that all provided path arguments are absolute?

Reply:
I will create a separate issue for this. Let's not hold up your PR any further.